tests/ui/match/match-float.rs

```diff
@@ -1,11 +1,48 @@
 //@ run-pass
 // Makes sure we use `==` (not bitwise) semantics for float comparison.
 
-fn main() {
+#![feature(f128)]
+#![feature(f16)]
+
+fn check_f16() {
+    const F1: f16 = 0.0;
+    const F2: f16 = -0.0;
+    assert_eq!(F1, F2);
```

Comment:
You got an error here. Assuming you ran the tests on your machine, is this platform-specific behavior? Sounds like a nightmare

Reply:
That said, this is unrelated to this PR since this PR only deals with exhaustiveness checking of floats. You could comment these tests out and fix that in a different PR.

Reply:
Ah yeah if these are hitting any runtime tests then there will be problems on some platforms, I thought everything was going through const eval. I’ll cfg the relevant parts out when I’m back at a computer.